bootstrap.sh

```diff
@@ -114,7 +114,12 @@ create_symlinks() {
     lnif "$source_path/.vimrc.bundles" "$target_path/.vimrc.bundles"
     lnif "$source_path/.vimrc.before"  "$target_path/.vimrc.before"
     lnif "$source_path/.vim"           "$target_path/.vim"
-
+    
+    if has('nvim')
```

Comment:
I would still like to use `has('nvim')` in combination with an environment variable to specify that nvim is your only vim, but not as the default.